src/main/scala/dpla/batch_process_dpla_index/helpers/Index.scala

```diff
@@ -73,4 +73,93 @@ class Index(
     }
     replResponse.close()
   }
+
+  def deleteIndex(): Unit = {
+    val request: Request =
+      new Request.Builder()
+        .url(s"http://$host:$port/$indexName")
+        .delete()
+        .build()
+    val response: Response = httpClient.newCall(request).execute()
+    if (!response.isSuccessful) {
+      throw new RuntimeException(
+        s"FAILED request for ${request.url.toString} (${response.code}, " +
+          s"${response.message})"
+      )
+    }
+    System.out.println(s"Deleted $indexName")
+    response.close()
```

Comment:
is response something that needs to be cleaned up? should it be in a finally block or something?

Reply:
Yes, thank you!  I have moved all of the `resopnse.close()` calls into `finally` blocks: https://github.com/dpla/batch-process-dpla-index/pull/10/commits/2aa4e1db66ce7d86724fa07ef486e17c5316a766